src/SamplesApp/SamplesApp.UITests/Windows_UI_Xaml_Controls/ListViewTests/UnoSamples_Tests.ListView.cs

```diff
@@ -163,5 +158,166 @@ public void ListView_ObservableCollection_Unused_Space()
 
 			Assert.AreEqual(3 * heightBefore, heightAfter);
 		}
+		
+		[Test]
+		[AutoRetry]
+		public void ListView_ExpandableItem_ExpandSingleItem()
+		{
+			Run("SamplesApp.Windows_UI_Xaml_Controls.ListView.ListView_Expandable_Item");
+
+			var checkBox = _app.Marked("CheckBox");
+			_app.WaitForElement(checkBox);
+
+			// Save initial state(not expanded)
+			var screenshot1 = _app.Screenshot("ListView Expanded Control - Initial State");
+
+			// Expand and compare
+			ClickCheckBoxAt(0);
+			var screenshot2 = _app.Screenshot("ListView Expanded Control - expanded State");
+			if (!UITestHelper.CompareScreenShots(screenshot1, screenshot2))
+			{
+				Assert.Fail("Expand is not working");
+			}
+
+			// Collapse and compare
+			ClickCheckBoxAt(0);
+			var screenshot3 = _app.Screenshot("ListView Expanded Control - collapsed State");
+			if (UITestHelper.CompareScreenShots(screenshot1, screenshot3))
+			{
+				Assert.Fail("Collapse is not working");
+			}
+		}
+
+		[Test]
+		[AutoRetry]
+		public void ListView_ExpandableItem_ExpandMultipleItems()
+		{
+			Run("SamplesApp.Windows_UI_Xaml_Controls.ListView.ListView_Expandable_Item");
+
+			var checkBox = _app.Marked("CheckBox");
+			_app.WaitForElement(checkBox);
+
+			// Save initial state(not expanded)
+			var screenshot1 = _app.Screenshot("ListView Expanded Control - Initial State");
```

Comment:
Use `TakeScreenShot` instead, which does not need prefixing with the name of the test.